source/arche/tutorial/testing.rst

```diff
@@ -73,18 +73,11 @@ You can then test the test's initial condition
     EXPECT_DOUBLE_EQ(facility->input.quantity(), qty);
     EXPECT_DOUBLE_EQ(facility->inventory.quantity(), 0);
 
-Now your test should look like
+Then execute the Tock:
 
 .. code-block:: c++
 
-    TEST_F(StorageTest, Tock) {
-      double qty = 42;
-      facility->input.Push(cyclus::NewBlankMaterial(qty));
-      EXPECT_DOUBLE_EQ(facility->input.quantity(), qty);
-      EXPECT_DOUBLE_EQ(facility->inventory.quantity(), 0);
       EXPECT_NO_THROW(facility->Tock());
-      // Test Storage specific behaviors of the Tock function here
-    }
 
```

Comment:
That is a documentation update which has not reached the stubs yet. FYI.

On Mon, May 18, 2015 at 2:10 PM, vcloitre <notifications@github.com> wrote:

> In source/arche/tutorial/testing.rst
> <https://github.com/gidden/cyclus.github.com/pull/3#discussion_r30538076>:
>
> >        EXPECT_NO_THROW(facility->Tock());
> > -      // Test Storage specific behaviors of the Tock function here
> > -    }
> >
>
> I had to replace all "facility" with "src_facility_" in the test for the
> Tock function or else the build failed. Was it working for you with
> "facility" @mbmcgarry <https://github.com/mbmcgarry> ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gidden/cyclus.github.com/pull/3/files#r30538076>.
>



-- 
Matthew Gidden, Ph.D.
Postdoctoral Associate, Nuclear Engineering
The University of Wisconsin -- Madison
Ph. 225.892.3192